firefox-ios/nimbus-features/messaging/messaging.fml.yaml

```diff
@@ -182,6 +187,20 @@ objects:
           before it is expired.
         default: 5
 
+  MicrosurveyConfig:
+    description: >
+        Attributes relating to microsurvey messaging.
+    fields:
+      target-feature:
+        type: TargetFeature
+        description: >
+            The type of the feature the survey is targeted for.
+        default: Unknown # Should not be defaulted
+      options:
+        description: The list of survey options to present to the user.
+        type: List<String>
+        default: [] # Should not be defaulted
```

Comment:
tom - instead of putting the harded-coded options here, I made the default [] to match the other configs; happy to revert if that's preferred

Reply:
Yep, this is the right route, we defaulted here, and I can see it's going to cause a problem as `[List](https://experimenter.info/fml-spec#structural-types)` cannot be merged. Android will follow suit.

Reply:
ah, thanks for linking that, sounds good!